pandas-stubs/io/common.pyi

```diff
@@ -17,15 +17,7 @@ def validate_header_arg(header) -> None: ...
 def stringify_path(
     filepath_or_buffer: FilePathOrBuffer[AnyStr],
 ) -> FilePathOrBuffer[AnyStr]: ...
-def is_s3_url(url) -> bool: ...
-def is_gcs_url(url) -> bool: ...
 def urlopen(*args, **kwargs) -> IO: ...
-def get_filepath_or_buffer(
-    filepath_or_buffer: FilePathOrBuffer,
-    encoding: str | None = ...,
-    compression: str | None = ...,
-    mode: str | None = ...,
-): ...
```

Comment:
I have a feeling that this entire file can be deleted, because it has nothing that is public in it.

Reply:
I agree. Happy to delete it - wanted to avoid "controversial" changes.